src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -439,39 +439,19 @@ private async Task AddMcPkgsAsync(Invitation invitation, IList<string> mcPkgScop
         {
             var meeting = await _meetingClient.CreateMeetingAsync(meetingBuilder =>
             {
+                var baseUrl = _meetingOptions.CurrentValue.PcsBaseUrl +
```

Comment:
Use string interpolation + ensure this to work if config misses the / at end of csBaseUrl:
var baseUrl = $"{_meetingOptions.CurrentValue.PcsBaseUrl.Trim('/')}/{_plantProvider.Plant.Substring(4, _plantProvider.Plant.Length - 4).ToUpper()}";